src/test/java/com/ericsson/ei/flowtests/TestConfigs.java

```diff
@@ -0,0 +1,106 @@
+package com.ericsson.ei.flowtests;
```

Comment:
This file is used by many tests, not only in flow tests, please revert changes and move it back to test.utils

Reply:
done